tests/test_parser.py

```diff
@@ -257,6 +259,98 @@ def test_convert_to_date_cast_result_to_date(self, mocked_locale,
                          expected_output)
 
 
+class Test_bs_safeget(unittest.TestCase):
+    def test_bs_safeget_return_expected_text(self):
+        mocked_soup = MagicMock()
+        selector = 'some_tag'
+        mocked_soup.select.return_value = ['some text']
+        self.assertEqual(bs_safeget(mocked_soup, selector), 'some text')
+
+    def test_bs_safeget_return_false_when_selector_is_missing(
+            self):
+        mocked_soup = Mock()
+        mocked_soup.select.return_value = ''
+        selector = 'h1'
+        self.assertFalse(bs_safeget(mocked_soup, selector))
+
+
+@patch('smarsy.parse.bs_safeget')
+class TestCreateParentsDict(unittest.TestCase):
+    def test_bs_safeget_called_with_expected_html_for_img_getting(
+            self, mocked_get):
+        tag1, tag2 = '[valign=top]', 'img[src]'
+        create_parents_dict('html')
+        mocked_get.assert_called_with('html', tag1, tag2)
+
+    def test_create_parents_dict_return_false_with_none_img(self, mocked_get):
+        mocked_get.return_value = None
+        self.assertFalse(create_parents_dict('html'))
+
+    @patch('builtins.hasattr')
+    def test_bs_safeget_return_expected_img(self, mock_hasattr, mocked_get):
```

Comment:
mocked_get? copy/paste?

Reply:
Don't understand

Reply:
why hasattr mock is named as `mocked_get`?

Reply:
two mocks
@patch('smarsy.parse.bs_safeget') as Class Mock
@patch('builtins.hasattr')
and 
two parameters
mock_hasattr, mocked_get

Reply:
two mocks
@patch('smarsy.parse.bs_safeget') as Class Mock
@patch('builtins.hasattr')
and 
two parameters

Reply:
I'm asking why is the mock named as mocked_**get**